### PR TITLE
Refactor folder structure

### PR DESCRIPTION
**Pull request by Charles-Gagnon**

### CODE REVIEW

**Comment by chlafreniere** [comment]
@Charles-Gagnon can you please provide a description on what the motivation behind the refactor is?

In addition, looks like the integration tests failed.

File: src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj [hunk 1]
```diff
@@ -12,20 +12,27 @@
...
     <DebugType>embedded</DebugType>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Sql</PackageId>
+    <RootNamespace></RootNamespace>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (chlafreniere) [inline src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:22]: what's this newfolder stuff?

Comment (Charles-Gagnon) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:22]: hah, oops. Was messing around with stuff and apparently VS didn't clean this up as nicely as I thought

Comment (chlafreniere) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:22]: just saw the property above too. coul dyou take a look?

Comment (Charles-Gagnon) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj]: Yeah the root namespace? I added a comment explaining that. I think it's a bug with the analyzer or something...for files in the root of the project folder it always gives an error about the namespace.

I'll probably restructure them to be in sub-folders (which fixes that issue) in the future but for now this seems fine enough.

**Pull request merged**